goldenmaster/examples/olinkclient/main.cpp

```diff
@@ -39,62 +39,84 @@
+    client.linkObjectSource(tbSame2SameEnum1Interface);
+    tb_same2::SameEnum1InterfaceTraced tbSame2SameEnum1InterfaceTraced(tbSame2SameEnum1Interface );
     auto tbSame2SameEnum2Interface = std::make_shared<tb_same2::OLinkSameEnum2Interface>();
-    client.linkObjectSource(tbSame2SameEnum2Interface);tb_same2::SameEnum2InterfaceTraced tbSame2SameEnum2InterfaceTraced(tbSame2SameEnum2Interface );
+    client.linkObjectSource(tbSame2SameEnum2Interface);
+    tb_same2::SameEnum2InterfaceTraced tbSame2SameEnum2InterfaceTraced(tbSame2SameEnum2Interface );
     auto tbSimpleSimpleInterface = std::make_shared<tb_simple::OLinkSimpleInterface>();
-    client.linkObjectSource(tbSimpleSimpleInterface);tb_simple::SimpleInterfaceTraced tbSimpleSimpleInterfaceTraced(tbSimpleSimpleInterface );
+    client.linkObjectSource(tbSimpleSimpleInterface);
+    tb_simple::SimpleInterfaceTraced tbSimpleSimpleInterfaceTraced(tbSimpleSimpleInterface );
     auto tbSimpleSimpleArrayInterface = std::make_shared<tb_simple::OLinkSimpleArrayInterface>();
-    client.linkObjectSource(tbSimpleSimpleArrayInterface);tb_simple::SimpleArrayInterfaceTraced tbSimpleSimpleArrayInterfaceTraced(tbSimpleSimpleArrayInterface );
+    client.linkObjectSource(tbSimpleSimpleArrayInterface);
+    tb_simple::SimpleArrayInterfaceTraced tbSimpleSimpleArrayInterfaceTraced(tbSimpleSimpleArrayInterface );
     auto testbed1StructInterface = std::make_shared<testbed1::OLinkStructInterface>();
-    client.linkObjectSource(testbed1StructInterface);testbed1::StructInterfaceTraced testbed1StructInterfaceTraced(testbed1StructInterface );
+    client.linkObjectSource(testbed1StructInterface);
+    testbed1::StructInterfaceTraced testbed1StructInterfaceTraced(testbed1StructInterface );
     auto testbed1StructArrayInterface = std::make_shared<testbed1::OLinkStructArrayInterface>();
-    client.linkObjectSource(testbed1StructArrayInterface);testbed1::StructArrayInterfaceTraced testbed1StructArrayInterfaceTraced(testbed1StructArrayInterface );
+    client.linkObjectSource(testbed1StructArrayInterface);
+    testbed1::StructArrayInterfaceTraced testbed1StructArrayInterfaceTraced(testbed1StructArrayInterface );
 
-    QQmlApplicationEngine engine;
-    const QUrl url(QStringLiteral("qrc:/main.qml"));
-    QObject::connect(&engine,
-                     &QQmlApplicationEngine::objectCreated,
-                     &app,
-                     [url](QObject *obj, const QUrl &objUrl) {
-                       if (!obj && url == objUrl)
-                        QCoreApplication::exit(-1);
-                     },
-                     Qt::QueuedConnection);
-    engine.load(url);
+    
+    // Try out properties: subscribe for changes
+    testbed2ManyParamInterfaceTraced.connect(&testbed2ManyParamInterfaceTraced, &testbed2::AbstractManyParamInterface::prop1Changed, []( int prop1){ qDebug() << "prop1 property changed ";});

```

Comment:
looks like we are only using testbed2 and not iterating over all interfaces as above?